powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/PowerFactoryImporter.java

```diff
@@ -376,6 +380,7 @@ private void createSwitch(ImportContext importContext, VoltageLevel vl, MutableI
             throw new PowsyblException("Multiple staSwitch not supported");
         }
         DataObject staSwitch = staSwitches.isEmpty() ? null : staSwitches.get(0);
+        int busIndexIn = staCubic.getIntAttributeValue("obj_bus");
```

Comment:
Are we sure obj_bus is always defined?

Reply:
No, I am not sure, but I have the same problem with almost all the parameters. I have decided to manage a parameter as optional only when  I have a case where this parameter is not defined. By the moment  the `obj_bus` parameter is defined in all the cases that I have.

Reply:
Ok fine. Yes this is a major difficulty with such a loose model. But maybe worst it is very difficult to known from which version of power Factory an attribute has been added or replace another one and so on to state clearly which version we are supporting